README.md

```diff
@@ -72,7 +72,15 @@ black .
 
 ### `/listeners`
 
-Every incoming request is routed to a "listener". Inside this directory, we group each listener based on the Slack Platform feature used, so `/listeners/events` handles incoming events, `/listeners/shortcuts` would handle incoming [Shortcuts](https://api.slack.com/interactivity/shortcuts) requests, and so on.
+Every incoming request is routed to a "listener". Inside this directory, we group each listener based on the Slack Platform feature used, so `/listeners/events` handles incoming events like messages sent to the app.
```

Comment:
👁️‍🗨️ quibble: This line might be good to keep for hypothetical setups? Though that link ought be perhaps to https://docs.slack.dev/interactivity/implementing-shortcuts/ if so...